Hardware/Computer.cs

```diff
@@ -94,19 +97,25 @@ public class Computer : IComputer {
         Add(new RAM.RAMGroup(smbios, settings));
 
       if (gpuEnabled) {
-        Add(new ATI.ATIGroup(settings));
+        //Add(new ATI.ATIGroup(settings));
         Add(new Nvidia.NvidiaGroup(settings));
       }
 
       if (fanControllerEnabled) {
-        Add(new TBalancer.TBalancerGroup(settings));
+        //Add(new TBalancer.TBalancerGroup(settings));

```

Comment:
Should be undone.

Reply:
Sorry about this -> Thats clearly wrong -> Changed